### PR TITLE
Hide stickied post and "It feels lonely in here" when subverse is sorted by "top".

### PR DESCRIPTION
**Pull request by gamehelp16**
But the "It feels lonely in here" thing will be replaced by a little message.

voattop

(Also sorry for the "weird" first commit, I tried to update my fork using the "pull request" method because I don't use GitHub's application)

### CODE REVIEW

**Comment by gamehelp16** [comment]
Hello?

**Comment by DanielFlamino** [comment]
This tends to take some time sometimes. My recommendation is patience.

**Comment by gamehelp16** [comment]
Oh, okay.

I'm just wondering because voat has just commited something too recently.

**Comment by voat** [comment]
Thank you for this PR. Instead of adding a new CSS class (note that we have both light and dark themes), could you please use one of the existing classes, or perhaps something like this as the error message is in fact an alert/notification:

```html
<div class="alert alert-info">
<p>
There are no links or discussions for selected time period.
</p>
</div>
```

**Comment by gamehelp16** [comment]
Done :)

File: Whoaverse/Whoaverse/Views/Subverses/SubverseIndex.cshtml [hunk 1]
```diff
@@ -89,8 +92,17 @@
 
             @if (Model.Count < 1)
             {
-                Html.RenderPartial("_DefaultMessage", new ViewDataDictionary() {{"selectedSubverse", selectedSubverse}});
-            }
+                @if (sortingMode == "top")
```
Comment (voat) [inline Whoaverse/Whoaverse/Views/Subverses/SubverseIndex.cshtml]: The @ character here is not allowed as we're already in a code block and compile fails. Compiler output:  Unexpected "if" keyword after "@" character.  Once inside code, you do not need to prefix constructs like "if" with "@".

**Comment by gamehelp16** [comment]
Done! (again) ;)

**Comment by PuttItOut** [comment]
Closing because conflicts.

**Pull request closed without merging**